sensu-plugins-zendesk.gemspec

```diff
@@ -21,23 +21,23 @@ Gem::Specification.new do |s|
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 2.0.0'
+  s.required_ruby_version  = '>= 2.1.0'
```

Comment:
As mentioned in the other PR, this needs to be called out in the changelog under `### Breaking Changes` so that we inform users on the impact and make it more obvious to reviewers and maintainers so it can be evaluated and versioned appropriately.